tests/10-vc-di.js

```diff
@@ -43,7 +43,7 @@ describe('Verify Credential - Data Integrity', function() {
       let validVc;
       before(async function() {
         const issuer = matchingIssuers.get(issuerName).issuers.find(
-          issuer => issuer.tags.has('Ed25519Signature2020'));
+          issuer => issuer.tags.has('vc-api'));
```

Comment:
Was this just incorrect earlier? Seems different in intent/topic from the rest of the PR, no?

I've no objections to the change--and no need to extract this or anything--just curious.